scripts/deployment/globals_mainnet.json

```diff
@@ -1 +1,42 @@
-{"contractVerification":true,"useLedger":true,"derivationPath":"m/44'/60'/2'/0/0","providerName":"mainnet","networkURL":"https://eth-mainnet.g.alchemy.com/v2/","blockscoutURL":"https://eth.blockscout.com","chainId":"1","gnosisSafeAddress":"0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552","gnosisSafeProxyFactoryAddress":"0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2","multiSendCallOnlyAddress":"0x40A2aCCbd92BCA938b02010E17A5b8929b49130D","timelockAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","baseURI":"https://gateway.autonolas.tech/ipfs/","componentRegistryName":"Component Registry","componentRegistrySymbol":"AUTONOLAS-COMPONENT-V1","agentRegistryName":"Agent Registry","agentRegistrySymbol":"AUTONOLAS-AGENT-V1","serviceRegistryName":"Service Registry","serviceRegistrySymbol":"AUTONOLAS-SERVICE-V1","componentRegistryAddress":"0x15bd56669F57192a97dF41A2aa8f4403e9491776","agentRegistryAddress":"0x2F1f7D38e4772884b88f3eCd8B6b9faCdC319112","registriesManagerAddress":"0x9eC9156dEF5C613B2a7D4c46C383F9B58DfcD6fE","serviceRegistryAddress":"0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA","serviceManagerAddress":"0x38b062d11CD7596Ab5aDFe4d0e9F0dC3218E5389","gnosisSafeMultisigImplementationAddress":"0x46C0D07F55d4F9B5Eed2Fc9680B5953e5fd7b461","gnosisSafeSameAddressMultisigImplementationAddress":"0xfa517d01DaA100cB1932FA4345F68874f7E7eF46","operatorWhitelistAddress":"0x42042799B0DE38AdD2a70dc996f69f98E1a85260","serviceRegistryTokenUtilityAddress":"0x3Fb926116D454b95c669B6Bf2E7c3bad8d19affA","serviceManagerTokenAddress":"0x2EA682121f815FBcF86EA3F3CaFdd5d67F2dB143","treasuryAddress":"0xa0DA53447C0f6C4987964d8463da7e6628B30f82","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","olasAddress":"0x0001A500A6B18995B03f44bb040A5fFc28E45CB0","minStakingDepositLimit":"10000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"0x4503b79d468e81Ad2d4bd6Db991810da269bA777","stakingFactoryAddress":"0xEBdde456EA288b49f7D5975E7659bA1Ccf607efc"}
```

Comment:
@kupermind I propose we don't have the left hand side formatting ever. It's annoying to review this way.

Reply:
Agree, with new deployment scripts it's never the case anymore.